### ISSUE TITLE
Add a Gitter chat 

### ISSUE THREAD

**Issue by naveensrinivasan** [octokit/octokit.net#926]
I would help us to have conversations in gitter https://gitter.im/.

**Comment by Haacked** [comment]
:+1:

**Comment by alfhenrik** [comment]
:+1:

**Comment by khellang** [comment]
Y NO SLACK?

**Comment by M-Zuber** [comment]
I actually kinda want both. They can serve different kinds of discussions

**Comment by shiftkey** [comment]
I'm kinda indifferent on which we choose (I neglect both really equally, so I don't really have a horse in the race) but I think sticking to one would be better.

Are there specific features from either platform you're really attached to?

**Comment by khellang** [comment]
Phone, phone, phone -> Slack :wink: Also, Gitter is kinda buggy :cry:

**Comment by khellang** [comment]
The only thing that suck with Slack is the invition model, but we recently set up slackin for Nancy and it works like a charm.

Just spin up https://github.com/rauchg/slackin and give it an API key.

**Comment by shiftkey** [comment]
Yep, I don't mind doing that to take away that invitation pain...

**Comment by alfhenrik** [comment]
Out of the two, I like Slack better, but I'm :cool: with either.

**Comment by M-Zuber** [comment]
I enjoy Slack so don't mind Slack only. Gitter on the phone is a bit of a pain, but it's still better the Github itself :wink:
The main selling point of gitter for me is the integration. If Slack has something similar then I'll withdraw my gitter vote.

**Comment by khellang** [comment]
@M-Zuber What kind of integration are you looking for?

**Comment by M-Zuber** [comment]
Now that my computer started listening to me again ( I swear to God some days it behaves worse then a two year old), I was able to check the Slack integration with Github and it looks nice.
The main difference is in Gitter the information is displayed in a dedicated side bar, while Slack treats each event as just another message.

tl;dr I've been converted to Slack

**Comment by shiftkey** [comment]
Hearing news like [this](https://github.com/reactiflux/volunteers/issues/17#issuecomment-147300333) concerns me a bit:

> I got on the phone with a couple of Slack people last week, and **they have decided not to support infinitely large public communities.** Slack wants to focus on building team communication software, and groups like Reactiflux don't fit into that.
> ...
> **The only way we could stay with Slack would be to cap our user base at < 8k. Probably around 1-3K. This obviously doesn't work for an open community.** The most important thing about our community is that it's open and anyone can join in less than a minute. So when new people want to join the conversation, it's very low friction. We obviously can't keep the friction low if we start moderating accounts to keep our numbers low.

I know we wouldn't be at that scale for a while, but it's in the back of my mind.

cc @pengwynn

**Comment by alfhenrik** [comment]
That's a bit unfortunate...I like Slack a lot, but if that's their stance, then perhaps Gitter would be a better alternative...

**Comment by M-Zuber** [comment]
https://sameroom.io/blog/connecting-a-channel-in-slack-to-a-room-in-gitter/
https://sameroom.io/pricing
I am not sure how the pricing would work, but maybe there is a similar free option.

**Comment by ferventcoder** [comment]
Slack also caps search history... If it weren't for that, slack would be nice.

**Comment by ferventcoder** [comment]
I tend to grab urls from gitter discussions to use in issues and emails.

**Comment by M-Zuber** [comment]
There is also a new player in town: http://www.ryver.com/
I have played with it a bit and it is quite nice actually

**Comment by M-Zuber** [comment]
Re capping the search history, I just saw this:http://techcrunch.com/2015/11/30/slackarchive-gives-you-public-chat-archive-for-free/
@ferventcoder

**Comment by ferventcoder** [comment]
Looks interesting, but not built in so it presents a point of failure. Plus you can't see the entire conversation surrounding it, only messages that you search for. Not exactly apples and apples. :)

**Comment by M-Zuber** [comment]
True. Just thought it interesting. 
I anyways strongly recommend gitter. 
Ryvver is nice for teams, but I don't see any integration points right now

**Comment by ferventcoder** [comment]
I saw ryver a few weeks ago. I agree that it looks interesting. We'll see how they do given how nice slack is.

**Comment by M-Zuber** [comment]
I think no integrations will make it rough. But in my interactions with them,  they have proven very open to suggestions and make changes quickly

**Comment by M-Zuber** [comment]
Ryvver has zapier and OData for integrations. Not super exciting

**Comment by khellang** [comment]
There's also https://rocket.chat/ :wink:

**Comment by naveensrinivasan** [comment]
@shiftkey / @Haacked  Could we create one of these options?

**Comment by Haacked** [comment]
I vote for Gitter simply because github/VisualStudio is already there and I want to avoid a million different chat systems. :stuck_out_tongue:

**Comment by Haacked** [comment]
https://gitter.im/octokit/octokit.net

DONE!
